.buildkite/pipeline.build_py37.yml

```diff
@@ -2,62 +2,64 @@
 
 # These tests install requirements_legacy_compat.txt which are the
 # lower bound of dependencies we support.
-- label: ":cold_face: :python: Ray Python 3.7 legacy dependency ML compatibility tests"
-  conditions:
-    ["RAY_CI_PYTHON_DEPENDENCIES_AFFECTED", "RAY_CI_TUNE_AFFECTED", "RAY_CI_TRAIN_AFFECTED", "RAY_CI_ML_AFFECTED"]
-  instance_size: large
-  parallelism: 3
-  commands:
-    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
-    - ./ci/env/install-minimal.sh 3.7
-    - PYTHON=3.7 DATA_PROCESSING_TESTING=1 TUNE_TESTING=1 TRAIN_TESTING=1 INSTALL_HDFS=1 ./ci/env/install-dependencies.sh
-    - pip install -r python/requirements/compat/requirements_legacy_compat.txt
-    - pip install -U typing-extensions
-    - HOROVOD_WITH_GLOO=1 HOROVOD_WITHOUT_MPI=1 HOROVOD_WITHOUT_MXNET=1 HOROVOD_WITH_TENSORFLOW=1 HOROVOD_WITH_PYTORCH=1 pip install horovod
-    - ./ci/env/env_info.sh
-    # Combine shards from different files
-    - >-
-      set -x;
-      {
-        python ./ci/ray_ci/bazel_sharding.py --exclude_manual --index "\${BUILDKITE_PARALLEL_JOB}" --count "\${BUILDKITE_PARALLEL_JOB_COUNT}" --tag_filters=compat python/ray/tests/horovod/... python/ray/tests/lightgbm/... python/ray/tests/ml_py37_compat/... python/ray/tests/xgboost/... &&
-        python ./ci/ray_ci/bazel_sharding.py --exclude_manual --index "\${BUILDKITE_PARALLEL_JOB}" --count "\${BUILDKITE_PARALLEL_JOB_COUNT}" --tag_filters=-gpu,-hdfs python/ray/air/... &&
-        python ./ci/ray_ci/bazel_sharding.py --exclude_manual --index "\${BUILDKITE_PARALLEL_JOB}" --count "\${BUILDKITE_PARALLEL_JOB_COUNT}" --tag_filters=ray_air,-torch_1_11,-gpu_only,-gpu,-hdfs,-new_storage python/ray/train/... &&
-        python ./ci/ray_ci/bazel_sharding.py --exclude_manual --index "\${BUILDKITE_PARALLEL_JOB}" --count "\${BUILDKITE_PARALLEL_JOB_COUNT}" --tag_filters=ray_air python/ray/data/...;
-      } > test_shard.txt
-    - cat test_shard.txt
-    - bazel test --config=ci $(./ci/run/bazel_export_options)
-      --test_env=RAY_AIR_NEW_PERSISTENCE_MODE=0
-      $(cat test_shard.txt)
+# TODO(justinvyu): Uncomment once the individual test suites are passing.
```

Comment:
Can you add more detail here? Why are these being commented out in this PR?

Reply:
Discussed offline - The tests in here overlap with other test suites, so we will migrate those test suites and then enable these.